pkg/plan/track_response.go

```diff
@@ -41,16 +41,12 @@ type TrackResponse struct {
 	runningStep bool
 }
 
-func (res TrackResponse) Error() error {
+func (res TrackResponse) Error() string {
```

Comment:
Will this change have any unintended consequences for the terraform provider?

Reply:
Not at all, the Terraform provider relies on the `Wait()` and the result it has, by default it returns the result of this, but if. we wanted to we could turn it into JSON. So long story short it only adds features which we can leverage ourselves.